lib/cli/writer.js

```diff
@@ -23,7 +23,7 @@ const os = require('os');
 const unmount = require('./unmount');
 const imageStream = require('../image-stream');
 const errors = require('../shared/errors');
-const constraints = require('../shared/constraints');
+const constraints = require('../shared/drive-constraints');
```

Comment:
Ooops, nasty typo in #1223 ? :-(
Does the fact that this wasn't caught mean that it wasn't being unit-tested properly? Could this PR add appropriate unit-tests?